src/VolleyManagement.Backend/VolleyManagement.Domain/Properties/Resources.resx

```diff
@@ -243,9 +243,15 @@
   <data name="ValidationTeamAchievements" xml:space="preserve">
     <value>Team achievements can not contain more than  {0} symbols</value>
   </data>
+  <data name="ValidationTeamCaptain" xml:space="preserve">
+    <value>Captain's id is wrong</value>
```

Comment:
what does it mean wrong? has wrong religious beliefs?

Reply:
Be more specific?